packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -134,48 +97,18 @@ export const Stackbox = () => {
   const { chain } = useNetwork();
   const { switchNetwork } = useSwitchNetwork({
     onSuccess(data) {
-      setChainId(data.id);
+      handleChainIdChange(data.id);
```

Comment:
is there a reason to change from `setChainId`, it's seems more consistent with the rest of things.

Reply:
As defined in an offline chat, will rename this function into `setChainId` as it would be an expected behavior to reset the form values when changing the chain.